src/views/ingresses/details/tabs/details/components/IngressDetailsSection/IngressDetailsSection.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react';
+
+import { IngressModel, NamespaceModel } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiNetworkingV1Ingress } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import {
+  getGroupVersionKindForModel,
+  ResourceLink,
+  Timestamp,
+  useAccessReview,
+  useAnnotationsModal,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { DetailsItem } from '@utils/components/DetailsItem/DetailsItem';
+import { LabelList } from '@utils/components/DetailsItem/LabelList';
+import { OwnerReferences } from '@utils/components/OwnerReference/owner-references';
+import Title from '@utils/components/Title/Title';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import {
+  getCreationTimestamp,
+  getLabels,
+  getName,
+  getNamespace,
+  getUID,
+} from '@utils/resources/shared';
+
+import TLSCert from './TLSCert';
+
+type IngressDetailsSectionProps = {
+  ingress: IoK8sApiNetworkingV1Ingress;
+};
+
+const IngressDetailsSection: FC<IngressDetailsSectionProps> = ({ ingress }) => {
+  const { t } = useNetworkingTranslation();
+  const annotationsModalLauncher = useAnnotationsModal(ingress);
+
+  const [canUpdate] = useAccessReview({
+    group: IngressModel?.apiGroup,
+    name: getName(ingress),
+    namespace: getNamespace(ingress),
+    resource: IngressModel?.plural,
+    verb: 'patch',
+  });
+
+  return (
+    <div className="co-m-pane__body">
```

Comment:
It feels to me we could use some nice PF components here instead, too. Or not? 🤔 (same for more code changes I can see in this PR)

Reply:
Done!

Reply:
Done!